flavors/template-Exasol-all-python-3.8-cuda-conda/flavor_base/build_deps/packages/apt_get_packages

```diff
@@ -1,9 +1,9 @@
 coreutils|8.28-1ubuntu1
-locales|2.27-3ubuntu1.6
```

Comment:
what is the reason to go back to an older version?

Reply:
probably a error and this file should get deleted anyway